.github/workflows/ssi-authority-schema-registry-service.yml

```diff
@@ -0,0 +1,91 @@
+###############################################################
```

Comment:
I suggest to shorten the file, remove the SSI Authority & Schema Registry part.

Reply:
adjusted